SingularityUI/app/components/requests/Columns.jsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import Column from '../common/table/Column';
+import moment from 'moment';
+
+import Utils from '../../utils';
+
+import JSONButton from '../common/JSONButton';
+
+import UnpauseButton from './UnpauseButton';
+import RemoveButton from './RemoveButton';
+
+// use this only with combineStarredWithRequests selector
+export const Starred = (changeStar, sortable = false) => (
```

Comment:
Did you mean to use
```
export const Starred = ({changeStar, sortable}) => (
```
and defaultProps? I assume changeStart and sortable are properties for this react component.